TWLight/static/css/new-local.css

```diff
@@ -396,15 +407,15 @@ LOGIN PARTIAL CSS
         font-style: normal;
         padding: 35px 0px 0px 125px;
     }
-
-    .criteria-info{
-        font-size: 20px;
-    }
-    .criteria-item{
-        font-family: Roboto-Light;
-        font-size: 20px;
-        line-height: 32px;
-    }
+     .criteria-info{
```

Comment:
unnecessary whitespace change? actual question; the indentation may have been incorrect before.